mix.exs

```diff
@@ -42,7 +42,11 @@ defmodule Plug.MixProject do
       {:ex_doc, "~> 0.17.1", only: :docs},
       {:inch_ex, ">= 0.0.0", only: :docs},
       {:hackney, "~> 1.2.0", only: :test},
-      {:kadabra, "~> 0.3.4", only: :test}
+      {:kadabra, "~> 0.3.4", only: :test},
+
+      # Benchmarks
+      {:benchee, "~> 0.11.0"},
```

Comment:
I think this should be `only: :test`. Otherwise, projects depending on `plug` will also depend on `benchee` and `benchee_json`

Reply:
Oh, I see that.. the benchmarks are run with `MIX_ENV=bench`, so adding `only: :bench`  should be enough.